docs/Deployment-Procedure.md

```diff
@@ -170,3 +170,64 @@ The last step is to modify the frontend url file for the triage portal. Log in t
+| Website  | Database | Location |
+| ------------- | ------------- | ------------- |
+| cbioportal.mskcc.org  | cgds_gdac  | pipelines |
+| cbioportal.org  | cgds_public  | AWS |
+| genie.cbioportal.org | cgds_genie | AWS | 
+| triage.cbioportal.org | cgds_triage | pipelines |
+
+To obtain information such as usernames, passwords, hostnames - ask Avery, Angelica, Rob, Manda, and Ino. 
+
+## Updating Importers/Data Fetchers
+Importers (code found [here](https://github.com/knowledgesystems/pipelines)) and data fetchers (code found [here](https://github.com/knowledgesystems/cmo-pipelines)) use code from the cBioPortal codebase. The cbioportal dependency is packaged with the genome-nexus-annotation-pipeline and specified in the pipelines importer pom.
+
+The following steps are used during releases/updates to build new importers with the **most-up-to-date** cBioPortal and genome-nexus-annotation-pipeline code. All steps should be performed on the pipelines machine. 
+
+1. Set the jitpack hash [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline/blob/9510299395986653d9e9b672a38d472e52e7625b/pom.xml#L71) in the genome-nexus-annotation-pipeline codebase to the most recent cbioportal/cbioportal commit hash in master.
+
+2. Merge this change into genome-nexus-annotation-pipeline/master.
+
+3. Set the commit hash [here](https://github.com/knowledgesystems/pipelines/blob/f6c52bbda86b3929222d42c9bc84581fd6333fb4/pom.xml#L76) in the pipelines codebase to the most most recent genome-nexus/genome-nexus-annotation-pipeline commit hash **(after merge specfied in step 2)**. Also ensure the db version in the pom [here](https://github.com/knowledgesystems/pipelines/blob/f6c52bbda86b3929222d42c9bc84581fd6333fb4/pom.xml#L76) matches the db schema version in the cbiportal codebase. 
+
+4. Merge this change into pipelines/master.
+
+5. Set the commit hash [here](https://github.com/knowledgesystems/cmo-pipelines/blob/e740c9fa3d409ab75988e7a157682733e261fca5/cvr/pom.xml#L70) in the cmo-pipelines codebase to the most recent genome-nexus/genome-nexus-annotation-pipeline commit hash **(after merge specified in step 2)**
+
+6. Merge this change into cmo-pipelines/master
+
+7. Run the deployment wrapper script. See details [here](Deployment-Procedure.md#Deployment-Script).  
+
+8. Verify new importers/data fetchers have been placed in `/data/portal-cron/lib` by checking timestamps.
+```
+ls -tlra /data/portal-cron/lib
+```
+
+## Deployment Script
+The wrapper script is found on pipelines here:
+`/data/portal-cron/git-repos/pipelines-configuration/build-importer-jars/buildproductionjars.sh`.
+
+The wrapper script takes two arguments:
+1. --cbioportal-git-hash (required): Set to the cBioPortal commit hash being used in the pipelines build (hash specified in step 1 of [updating importers](#Updating-Importers). This must match because the build copies out resource files (e.g application-context-business.xml) from the cbioportal codebase. 
```

Comment:
I would boldface "step 1" .. to emphasize that the deployer should use the cbioportal codebase hash from the start of the procedure .. no the genome-nexus-annotation-pipeline hash (which they were working with most recently)